var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -144,6 +144,8 @@ class Cp2k(MakefilePackage, CMakePackage, CudaPackage, ROCmPackage):
         " conventional location. This option is only relevant when regtests need to be run.",
     )
 
+    variant("grpp", default=False, description="Enable GRPP psuedo potentials", when="@2025.2:")
```

Comment:
```suggestion
    variant("grpp", default=False, description="Enable GRPP psuedo potentials", when="@2025.2: build_system=cmake")
```
If I understand correctly, this is only supported with CMake?

Reply:
Yes this is correct. I forgot about the old build system. Thanks for pointing that out.